msteams-platform/concepts/build-and-test/share-to-teams-from-web-apps.md

```diff
@@ -21,19 +21,44 @@ The following image displays the pop-up window for Share to Teams preview experi
 
 You can also add link unfurling for the links shared through Share to Teams button hosted in web app, personal app or tab. For more information, see [link unfurling](~/messaging-extensions/how-to/link-unfurling.md).
 
-The following image displays the link unfurling experience through Share to Teams button:
-
-:::image type="content" source="~/assets/images/share-to-teams-link-unfurling.png" alt-text="Share-to-Teams link unfurling":::
-
```

Comment:
Why are we removing this?

Reply:
Reverted. Was done based on few changes from the initial understanding. Thanks!